inc/config.class.php

```diff
@@ -375,7 +377,11 @@ function showForm($options=[]) {
       echo "<td>";
       Dropdown::showYesNo("extradebug", $this->isFieldActive('extradebug'));
       echo "</td>";
-      echo "<td colspan=2></td>";
+
+      echo "<td>".__('Re-prepare jobs once finished', 'fusioninventory')."</td>";
```

Comment:
I would say 'By default, re-prepare successfull jobs'. What do you think about that ?

Reply:
I would say: 'Re-prepare successfull jobs'. The 'By default' amkes no sense because the check box nera the message indicates whether it is set or not

Reply:
ok let's go for Re-prepare successfull jobs !

Reply:
Updated and squashed